framerail/src/lib/server/load/login.ts

```diff
@@ -0,0 +1,44 @@
+import defaults from "$lib/defaults"
+import { translate } from "$lib/server/deepwell/translate"
+import type { TranslateKeys } from "$lib/types"
+import { parse } from "accept-language-parser"
+
+export async function loadLoginPage(request, cookies) {
+  // Set up parameters
+  const url = new URL(request.url)
+  const domain = url.hostname
+  const sessionToken = cookies.get("wikijump_token")
+  const language = request.headers.get("Accept-Language")
+  let locales = parse(language)
```

Comment:
Something I forgot to mention, but if the user has account-level preferences for locales, that would override whatever their local computer says (maybe they're logged in on a machine they don't typically use). Not sure where this step would go, since it would need to be fetched from the database.

Reply:
I would think there should be a store containing the user details that is only populated once for things like user profile in the top right corner, and we can pull the user locale preference from the store.